src/index.js

```diff
@@ -106,6 +107,15 @@ const settingsFeefines = [
   },
 ];
 
+const settingsPatronBlocks = [
+  {
+    route: 'conditions',
+    label: <FormattedMessage id="ui-users.settings.conditions" />,
+    component: ConditionsSettings,
+    // perm: 'ui-users.settings.conditions',
```

Comment:
Do we have implemented BE permissions? Why does it commented?

Reply:
Current permission contains subpermissions wich are already exists on BE.
Field was uncommented,